### PR TITLE
Selecting a branch to scan

### PR DESCRIPTION
**Pull request by mrfratello**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -178,8 +178,7 @@ export default async function main(
   threshold,
   repository,
   verbose,
-  maxContributors,
-  maxLostContributors
+  branch = "master"
```
Comment (onigoetz) [inline src/index.js:181]: shouldn't the default already be set from the cli param's defaults ?

**Review by onigoetz — COMMENTED** [review]
Hi, thanks for the PR, I'll test it tonight and if it's all good then merge and release

File: src/index.js (hunk 1 above)
Comment (mrfratello) [reply src/index.js]: agree, I deleted the extra default

**Comment by onigoetz** [comment]
Version 0.7.0 has been released and published to npm : https://github.com/onigoetz/absorption/releases/tag/0.7.0

**Pull request merged**